man/org.freedesktop.resolve1.xml

```diff
@@ -149,7 +149,6 @@ node /org/freedesktop/resolve1 {
       readonly s DNSStubListener = '...';
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
       readonly s ResolvConfMode = '...';
-      readonly b Monitor = ...;
```

Comment:
Maybe mention in the commit message that this was added in cb456374e096f0ebe9b70d7ddd98e16a4be24ee6 and can be dropped without backwards-compat concerns because no release yet.